etc/HIProdOfflineConfiguration.py

```diff
@@ -101,7 +101,7 @@
 # maxRunPreviousConfig = 999999 # Last run before era change 08/09/23
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_14_1_4_patch1",
+    'default': "CMSSW_14_1_4_patch4",
```

Comment:
```suggestion
    'default': "CMSSW_14_1_4_patch5",
```

given the outcome of https://github.com/dmwm/T0/pull/5017